docs/sources/api/_index.md

```diff
@@ -621,6 +621,17 @@ backing store. Mainly used for local testing.
 
 In microservices mode, the `/flush` endpoint is exposed by the ingester.
 
+### Tell ingester to release all resources on next SIGTERM
+
+```
+POST /ingester/prepare_shutdown
+```
+
+`/ingester/prepare_shutdown` will prepare the ingester to release resources on the next SIGTERM,
+where releasing resources means flushing data and unregistering from the ring.
```

Comment:
"unregistering from the ring"  Which ring?  Our documentation says we have six.  If we mean the ingester ring, let's make that clear.

Reply:
Yeah, the ingester ring. I pushed a commit addressing this.